src/packet/signature.js

```diff
@@ -591,25 +591,18 @@ Signature.prototype.toSign = function (type, data) {
     case t.cert_casual:
     case t.cert_positive:
     case t.cert_revocation: {
-      let packet;
-      let tag;
-
-      if (data.userid !== undefined) {
-        tag = 0xB4;
-        packet = data.userid;
-      } else if (data.userattribute !== undefined) {
-        tag = 0xD1;
-        packet = data.userattribute;
-      } else {
-        throw new Error('Either a userid or userattribute packet needs to be ' +
+      const packet = data.userId || data.userAttribute;
```

Comment:
Now that the userId and userAttributes are passed separately, the old signature code can be reverted right?

Reply:
Yes, with adjustment to the character case. I think I'll just remove the first commit completely and extract a test case from there.